mlrun/api/utils/builder.py

```diff
@@ -427,6 +497,14 @@ def build_image(
 
         runtime.spec.clone_target_dir = path.join(tmpdir, "mlrun", relative_workdir)
 
+    from mlrun.api.crud import Secrets
```

Comment:
why import is here?

Reply:
it caused circular import error when I moved it to the top of the file